.github/workflows/extract-translation-source-files.yml

```diff
@@ -181,21 +182,48 @@ jobs:
           repository: openedx/${{ matrix.repo }}
           path: translations/${{ matrix.repo }}
 
-      # Sets up Python
+      - name: prepare the environment with edx-platform specific changes
+        if: matrix.repo == 'edx-platform'
+        run: |
+          # This is needed for edx-platform base.txt packages
+          sudo apt install -y libxml2-dev libxmlsec1-dev libxslt1-dev
+
+          # Remove unwanted `en` django/djangojs po files
+          rm -rf translations/${{ matrix.repo }}/conf/locale/en
```

Comment:
Avoid committing translations from the repo into `openedx-translations` as shown in https://github.com/Zeit-Labs/openedx-translations/pull/87/files#diff-c8614aaf23732d5043dfb6b2d6ed36b66537ac5bad9feae7f21a022064358106:

```suggestion
          rm -rf translations/${{ matrix.repo }}/conf/locale/
```


<kbd>![image](https://github.com/openedx/openedx-translations/assets/645156/e01fe4ae-84e7-4f2d-b5d4-72009ee39115)</kbd>

Reply:
Sure! done, thank you